docs/data-sources/gateway_control_plane_list.md

```diff
@@ -0,0 +1,151 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_gateway_control_plane_list Data Source - terraform-provider-konnect"
+subcategory: ""
+description: |-
+  GatewayControlPlaneList DataSource
+---
+
+# konnect_gateway_control_plane_list (Data Source)
+
+GatewayControlPlaneList DataSource
+
+## Example Usage
+
+```terraform
+data "konnect_gateway_control_plane_list" "my_gatewaycontrolplanelist" {
+  filter = {
+    cloud_gateway = true
+    cluster_type = {
+      eq  = "...my_eq..."
+      neq = "...my_neq..."
+      oeq = "...my_oeq..."
+    }
+    id = {
+      eq  = "...my_eq..."
+      oeq = "...my_oeq..."
+    }
+    name = {
+      contains = "...my_contains..."
+      eq       = "...my_eq..."
+      neq      = "...my_neq..."
+      oeq      = "...my_oeq..."
```

Comment:
oeq doesn't seem to be supported for name https://docs.konghq.com/konnect/api/control-planes/latest/#/operations/list-control-planes

Reply:
Let me take a look today

Reply:
It was an error in a shared schema. Fixed in https://github.com/Kong/platform-api/commit/8f2cb1ce143adc33b9714577a2aaba2af9eb4782

Reply:
ah ok - i've raised a PR to update example for sorting - https://github.com/Kong/platform-api/pull/1199
Once merged, we can generate this again.
I've generated on local, the filter parameters are now fixed for name and cluster_type.